src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx

```diff
@@ -1,21 +1,73 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import {Box} from 'grid-emotion';
+
+import Link from '../../../components/link';
 
 import SettingsPageHeader from '../components/settingsPageHeader';
 import accountNotificationFields from '../../../data/forms/accountNotificationSettings';
 import Form from '../components/forms/form';
-import JsonForm from '../components/forms/jsonForm';
 
-const AccountNotifications = React.createClass({
+import FieldFromConfig from '../components/forms/fieldFromConfig';
+import Panel from '../components/panel';
+import PanelBody from '../components/panelBody';
+import PanelHeader from '../components/panelHeader';
+import PanelFooter from '../components/panelFooter';
+
+import IconChevronRight from '../../../icons/icon-chevron-right';
+
+export default class AccountNotifications extends React.Component {
   render() {
     return (
       <div>
         <SettingsPageHeader label="Notifications" />
         <Form>
-          <JsonForm location={this.props.location} forms={accountNotificationFields} />
+          <Box>
+            {accountNotificationFields.map(field => {
+              return <FormField key={field.title} field={field} />;
+            })}
+          </Box>
         </Form>
       </div>
     );
-  },
-});
+  }
+}
+
+class FormField extends React.Component {
+  static propTypes = {
+    field: PropTypes.object.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    let {title, fields, fineTuning} = this.props.field;
+
+    let linkStyle = {
```

Comment:
formatting-wise, what do we want to do with these objects? imo they're kinda noisy to have inline, could we put them at the bottom of the file maybe?

Reply:
also - would we use emotion here?

Reply:
It won't meet our linting rules if I just move it down (no-use-before-define)

Reply:
A styled component might make sense... but I'd prefer not to do that unless it's a more reusable thing